.travis.yml

```diff
@@ -28,11 +28,12 @@ install:
   - nvm install $NODE_VERSION
   - nvm use --delete-prefix $NODE_VERSION
   - npm install --global npm@6.2.0
-  - script/build --create-debian-package --create-rpm-package --compress-artifacts
+  - script/bootstrap
 
-script:
-  - script/lint
-  - script/test
+script: >
+  script/lint &&
```

Comment:
If linting fails, we want the build to halt immediately. In order to make that happen Travis CI, we need to combine all of the commands together. /xref https://github.com/travis-ci/travis-ci/issues/6188